pkgs/servers/mattermost/default.nix

```
@@ -37,6 +37,10 @@ buildGoModule rec {
     hash = "sha256-ZlvO/7kdMopIHBDdFp6wSQCR+NobGdDC6PcVd1iG16E=";
   };
 
+  # Makes nix-update-script pick up the fetchurl for the webapp.
+  # https://github.com/Mic92/nix-update/blob/master/nix_update/eval.py#L190
```

```suggestion
  # https://github.com/Mic92/nix-update/blob/dacee6fb5792ff7b671e8dee855772606c91dfc6/nix_update/eval.py#L190
```

nit: might be good to make this a permalink, since it refers to a specific line (and i feel linking to a potentially outdated version of `nix-update`, but to the correct place inside, is probably better than linking to the current version of `nix-update` in some random unhelpful place in case the code there has moved)

---

Fixed by linking to a tag.